buildSrc/src/main/kotlin/config-kotlin.gradle.kts

```diff
@@ -48,8 +48,10 @@ gradlePlugin {
 tasks.withType<KotlinCompile> {
     kotlinOptions {
         jvmTarget = "1.8"
```

Comment:
just saw this, does this matter? should we bump it?

Reply:
There's not really any benefit to bumping it, and it means you can technically build with just a JDK 8 installed, toolchains will download 16.

Reply:
I see